omnibus/Gemfile

```diff
@@ -1,6 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", github: ENV.fetch("OMNIBUS_GITHUB_REPO", "chef/omnibus"), branch: ENV.fetch("OMNIBUS_GITHUB_BRANCH", "main")
```

Comment:
Why we are removing the ENV.fetch?

Reply:
If you see the commit it says it's a temporary commit.

Reply:
We are not. If you see the commit it says it's a temporary commit